plugins/modules/eos_ntp_global.py

```diff
@@ -0,0 +1,1000 @@
+            all:
+              description: Service NTP requests received on any interface.
+              type: bool
+            access_lists:
+              description: Configure access control list.
+              type: list
+              elements: dict
+              suboptions:
+                afi:
+                  description: ip/ipv6 config commands.
+                  type: str
+                acls:
+                  description: Access lists to be configured under the afi
+                  type: list
+                  elements: dict
+                  suboptions:
+                    acl_name:
+                      description: Name of the access list.
+                      type: str
+                    direction:
+                      description: direction for the packets.
+                      type: str
+                      choices: ["in", "out"]
+                    vrf:
+                      description: VRF in which to apply the access control list.
+                      type: str
+        servers:
+          description: Configure NTP server to synchronize to.
+          type: list
+          elements: dict
+          suboptions:
+            vrf:
+              description: vrf name.
+              type: str
+            server:
+              description: Hostname or A.B.C.D or A:B:C:D:E:F:G:H.
+              type: str
+              required: True
+            burst:
+              description: Send a burst of packets instead of the usual one.
+              type: bool
+            iburst:
+              description: Send bursts of packets until the server is reached
+              type: bool
+            key:
+              description: Set a key to use for authentication.
+              type: int
+            local_interface:
+              description: Configure the interface from which the IP source address is taken.
+              type: str
+            maxpoll:
+              description: Maximum poll interval.
+              type: int
+            minpoll:
+              description: Minimum poll interval.
+              type: int
+            prefer:
+              description: Mark this server as preferred.
+              type: bool
+            version:
+              description: NTP version.
+              type: int
+        trusted_key:
+          description: Configure the set of keys that are accepted for incoming messages
+          type: str
+    running_config:
+      description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the EOS device by
+        executing the command B(show running-config | section ntp).
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into Ansible structured data as per the resource module's argspec
+        and the value is then returned in the I(parsed) key within the result.
+      type: str
+    state:
+      description:
```

Comment:
Please add a note stating that `state: overridden` and `state: replaced` are identical.